pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -288,6 +288,14 @@ def convert_nested_indexer(indexer_type, keys):
 
         tm.assert_series_equal(result, expected)
 
+    def test_multiindex_loc_one_dimensional_tuple(self, frame_or_series):
```

Comment:
can you parameterize over all of the selectors in the OP e.g. `obj.loc[('a',), :]` if its a DataFrame

Reply:
Added tests for df, I think one of the cases mentioned there should raise and not work